components/bl-line-chart-component/README.md

```diff
@@ -0,0 +1,40 @@
+| Width<br/>`width`       | Text                                                   | "500px"                                                                                                                                                                                                                                                                                                                                                                | Width Logic    | YES          | YES        | This handler allows you to specify the width of a component.                                                                                                                                                                                                                        |
+| Type<br/>`type`         | Select <br/> [Bar:`bar`<br/>Line:`line`<br/>Pie:`pie`] | Bar:`bar`                                                                                                                                                                                                                                                                                                                                                              | Type Logic     | YES          | YES        | This handler allows you to specify the chart type.                                                                                                                                                                                                                                  |
+| Labels<br/>`labels`     | JSON                                                   | `["A", "B", "C", "D"]`                                                                                                                                                                                                                                                                                                                                                 | Labels Logic   | YES          | YES        | This handler allows you to set chart elements. Watch [Codeless Examples](#Examples). Signature of labels: list of strings `[String, String]`                                                                                                                                        |
```

Comment:
The Codeless Examples link doesn't work in the browser. Try writing id in lowercase